app/src/main/res/layout/content_main.xml

```diff
@@ -80,7 +80,7 @@
                     android:layout_marginStart="50dp" />
 
                 <ToggleButton
-                    android:id="@+id/toggleMostRecent"
+                    android:id="@+id/toggleFriends"
```

Comment:
Swapped the positions of the "Most Recent" filter option and the "Friends Only" filter option. Looks much nicer now, especially when we hide the "Friends Only" option.